pom.xml

```diff
@@ -391,9 +397,13 @@
                 <extensions>true</extensions>
                 <configuration>
                     <instructions>
-                        <_exportcontents>io.cdap.plugin.kinesis.streaming.*;com.amazonaws.services.kinesis.*;
-                            org.apache.spark.streaming.kinesis.*;org.apache.spark.streaming.scheduler.*;
-                            com.amazonaws.*;com.amazonaws.protocol.json.*;
+                        <_exportcontents>
+                            io.cdap.plugin.kinesis.streaming.*;
+                            com.amazonaws.services.kinesis.*;
+                            org.apache.spark.streaming.kinesis.*;
```

Comment:
nit: Would it be possible to import only required files here?

Reply:
We tend to export the entire package to try to prevent any classloading issues from one class calling methods from another class and so on.